### PR TITLE
OSDOCS-2088: Added quick start code snippet syntax reference

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-2088
https://issues.redhat.com/browse/ODC-5653

Preview build: https://deploy-preview-33115--osdocs.netlify.app/openshift-enterprise/latest/web_console/creating-quick-start-tutorials?utm_source=github&utm_campaign=bot_dp#quick-starts-accessing-and-executing-code-snippets_creating-quick-start-tutorials

This targets 4.8

### CODE REVIEW

File: modules/quick-starts-accessing-and-executing-code-snippets.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+You can execute a CLI code snippet when it is included in a quick start from the web console. To use this feature, you must first install the Web Terminal Operator. The web terminal and code snippet actions that execute in the web terminal are not present if you do not install the Web Terminal Operator. Alternatively, you can copy a code snippet to the clipboard regardless of whether you have the Web Terminal Operator installed or not.
+
+[id="quick-starts-syntax-for-inline-code-snippets_{context}"]
+== Syntax for inline code snippets
+
+[source,yaml]
+----
+`code block`{{copy}}
+`code block`{{execute}}
```
Comment (sahil143) [inline modules/quick-starts-accessing-and-executing-code-snippets.adoc:16]: > `code block`{{execute}}

Just FYI, execute syntax has `Copy to Clipboard` action present implicitly i.e. if execute syntax is used copy to clipboard action will be present regardless of web terminal operator.

Comment (ahardin-rh) [reply modules/quick-starts-accessing-and-executing-code-snippets.adoc:16]: @sahil143 Thanks! I added a note

**Comment by ahardin-rh** [comment]
@ppitonak Can you please review? Thank you!

**Comment by ppitonak** [comment]
@ahardin-rh do you really want a review from me?

**Comment by ahardin-rh** [comment]
@gajanan-more Can you please provide a QE review? Thank you!

**Pull request merged**